api/editor/api-editor/src/main/java/ca/bc/gov/tno/ApiEditor.java

```diff
@@ -21,11 +21,11 @@
 		"ca.bc.gov.tno.azure",
 		"ca.bc.gov.tno.elastic",
 		"ca.bc.gov.tno.kafka" })
-public final class ApiEditor {
```

Comment:
`final` throws an error in Openshift, but is required to pass linting...  Not sure what I'll do to solve this when we enforce linting.